src/Dfc.CourseDirectory.Core/DataManagement/FileUploadProcessor.Courses.cs

```diff
@@ -366,6 +329,34 @@ protected async Task<UploadStatus> GetCourseUploadStatus(Guid courseUploadId)
             static bool RefMatches(string providerVenueRef, Venue venue) => providerVenueRef.Equals(venue.ProviderVenueRef, StringComparison.OrdinalIgnoreCase);
         }
 
+
```

Comment:
As above, this change should be backed out to keep the diff clean (this version is also missing the `Commit()`).

Reply:
I can't see the commit in the main branch. Could you give me a line number?